commit 67607f93b12e6564ad3f1c47324917266b2a07b5 (refs/changes/47/2647/5)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2018-01-11T19:48:22-08:00 (1 year, 9 months ago)
    
    MANTA-3443 Doc: Need more details on how to use Manta API as a subuser
    MANTA-3415 Doc: Specify units used for bandwidth in Manta Reports Reference
    Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>
    Approved by: Jordan Hendricks <jordan.hendricks@joyent.com>

diff --git a/docs/user-guide/rbac.md b/docs/user-guide/rbac.md
index 3a20671..85170ee 100644
--- a/docs/user-guide/rbac.md
+++ b/docs/user-guide/rbac.md
@@ -46,13 +46,23 @@ user names do not need to be globally unique.
 If there is a Triton account named `bigco` and another one named `littleco`,
 both can have a user named `contractor`.
 
+When making REST API requests as a user, simply specify the account in the
+`keyId` of the signature in the form of `account/user`. Specifically,
+
+an account keyId has this format:
+
+    bigco/keys/72:b1:da:b6:3f:3e:67:40:53:ca:c9:ab:0d:c4:2a:f7
+
+whereas a subuser's a keyId looks like this:
+
+    bigco/contractor/keys/a1:b2:c3:d4:e5:f6:a7:b8:c9:d0:e1:f2:a3:b4:c5:d6
+
 Manta tools use the following environment variables
 to make working with accounts and users easier.
 
 * `MANTA_USER` is the account owner.
 * `MANTA_SUBUSER` is a user within the account.
 
-
 **Roles** bring users, policies, and resources together.
 Roles define what users can do with a resource. For example, a role
 containing a policy with the `CAN getobject` action enables the members
@@ -70,6 +80,7 @@ Roles can be tagged to directories as well (the only exception is the root
 directory of the account). Directory access allows the user to list the objects
 in it but does not imply permissions for objects in it. This is different from
 the POSIX filesystems. In other words,
+
 - Roles on the directory are not required to access objects in that directory
   or to list the contents of subdirectories.
 - Roles tagged on a directory are not automatically cascaded to the objects or
@@ -77,11 +88,14 @@ the POSIX filesystems. In other words,
 
 When a user wants to access a resource,
 the access system checks whether the user
-belongs to a role associated with the resource.
+is a default member of role(s) associated with the resource.
 If so, the access system checks the policies
 associated with the role
 to determine whether the user can access the resource.
 
+If a specific role is passed in the request header (e.g. `-H "Role: operator"`),
+the access system will evaluate the permissions for that specific role only.
+
 The account owner always has complete access to every resource in the account.
 
 
diff --git a/docs/user-guide/reports.md b/docs/user-guide/reports.md
index 948c609..045448b 100644
--- a/docs/user-guide/reports.md
+++ b/docs/user-guide/reports.md
@@ -158,7 +158,7 @@ directory are counted under the first directory in this order: `stor` `public`
 ||**Field**||**Notes**||
 ||date||hour that this report is for||
 ||type||billable operation type, not HTTP method||
-||bandwidth||bandwidth in and out includes only successful GETs and PUTs||
+||bandwidth||bandwidth in and out, measured in bytes; includes only successful GETs and PUTs||
 
 
 # Compute Usage
